app/_src/gateway/install/index.md

```diff
@@ -15,7 +15,7 @@ disable_image_expand: true
   <p>
     <a href="/konnect/">{{ site.konnect_product_name }}</a> is an API lifecycle management platform that lets you build modern applications better, faster, and more securely.
   </p>
-  <p><a href="/konnect/getting-started/">Start for Free &rarr;</a></p>
+  <p><a href="https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=install-gateway">Start for Free</a></p>
```

Comment:
```suggestion
  <p><a href="https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=install-gateway" class="no-link-icon">Start for Free &rarr;</a></p>
```
Let's try this. It doesn't look quite as clean with the external link icon, and they had the arrow in the original design.